src/components/Tooltip/tests/Tooltip.test.tsx

```diff
@@ -33,6 +34,17 @@ describe('<Tooltip />', () => {
     );
   });
 
+  it('dismisses Tooltip once pointer is no longer over children, when preventInteraction is true', () => {
```

Comment:
I would write this more plainly as:
> passes `preventInteraction` to <TooltipOverlay />

And there should be a similar test within `TooltipOverlay.test.tsx` doing the same thing for `PositionedOverlay`